source/core_sntp_client.c

```diff
@@ -312,39 +321,48 @@ static SntpStatus_t sendSntpPacket( const UdpTransportInterface_t * pNetworkIntf
         {
             LogError( ( "Unable to send request packet: Transport send failed. "
                         "ErrorCode=%ld.", ( long int ) bytesSent ) );
-            sendError = true;
+            status = SntpErrorNetworkFailure;
         }
-
-        /* Partial sends are not supported by UDP, which only supports sending the entire datagram as a whole.
-         * Thus, if the transport send function returns status representing partial send, it will be treated as failure. */
-        else if( ( bytesSent > 0 ) && ( bytesSent != ( int32_t ) packetSize ) )
-        {
-            LogError( ( "Unable to send request packet: Transport send returned unexpected bytes sent. "
-                        "ReturnCode=%ld, ExpectedCode=%u", ( long int ) bytesSent, packetSize ) );
-
-            sendError = true;
-        }
-        else
+        else if( bytesSent == 0 )
         {
             /* No bytes were sent over the network. Retry send if we have not timed out. */
             SntpTimestamp_t currentTime;
 
             getTimeFunc( &currentTime );
 
             /* Calculate time elapsed since last data was sent over network. */
-            timeSinceLastSendMs = calculateElapsedTimeMs( &currentTime, &lastSendTime );
+            elapsedTimeMs = calculateElapsedTimeMs( &currentTime, &lastSendTime );
```

Comment:
`elapsedTimeMs` can also be scoped to this block

Reply:
Done